src/Discovery/DiscoveryListView.tsx

```diff
@@ -15,13 +15,15 @@ interface Props {
   setModalVisible: (boolean) => void;
   setModalData: (boolean) => void;
   selectedResources: any[];
+  selectedTags: any [];
   advSearchFilterHeight: string | number;
   setAdvSearchFilterHeight: (any) => void;
-  onResourcesSelected: (selectedResources: DiscoveryResource[]) => any
+  onResourcesSelected: (selectedResources: DiscoveryResource[]) => any;
+  onTagsSelected: (arg0: any) => any;
```

Comment:
```suggestion
  onTagsSelected: (selectedTags: any) => any;
```
better with a meaningful arg name